core/templates/pages/about-page/about-page.component.html

```diff
@@ -1,261 +1,670 @@
+        <p class="oppia-about-explore-lesson-content oppia-about-page-content" [innerHTML]="'I18N_ABOUT_PAGE_EXPLORE_LESSONS_CONTENT' | translate"></p>
+        <div class="oppia-about-classroom-button-container">
+          <button type="submit" (click)="onClickBrowseLibraryButton()"
+                  class="btn oppia-about-button oppia-about-button-browse"
+                  [innerHTML]="'I18N_ACTION_BROWSE_LIBRARY' | translate">
+          </button>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="oppia-about-page-section-six oppia-about-section">
+    <div class="oppia-section-six oppia-about-fixed-section">
+      <h2 class="oppia-about-outcome-title-mobile oppia-about-page-title-mobile" [innerHTML]="'I18N_ABOUT_PAGE_EASILY_CREATE_LESSON' | translate"></h2>
+      <div class="oppia-about-section-six-image-left">
+        <picture class="oppia-about-book-image">
+          <source type="image/webp" [srcset]="getStaticImageUrl('/about/books_dsk.webp')">
+          <source type="image/png" [srcset]="getStaticImageUrl('/about/books_dsk.png')">
+          <img [src]="getStaticImageUrl('/about/books_dsk.png')" alt="book image">
+        </picture>
+      </div>
+      <div class="oppia-about-section-six-right">
+        <h2 class="oppia-about-outcome-title oppia-about-page-title oppia-about-title-mobile-display-none" [innerHTML]="'I18N_ABOUT_PAGE_EASILY_CREATE_LESSON' | translate"></h2>
+        <p class="oppia-about-outcome-content oppia-about-page-content" [innerHTML]="'I18N_ABOUT_PAGE_CREATE_LESSON_CONTENT' | translate"></p>
+        <div class="oppia-about-classroom-button-container">
+          <button type="submit" (click)="onClickCreateLessonButton()"
+                  class="btn oppia-about-button oppia-about-button-browse"
+                  [innerHTML]="'I18N_ACTION_CREATE_LESSON_BUTTON' | translate">
+          </button>
         </div>
       </div>
     </div>
   </div>
-  <div class="oppia-footer-padding-below-banner">
+  <div class="oppia-about-page-section-seven oppia-about-section">
+    <h2 class="oppia-get-started-title oppia-about-page-title" [innerHTML]="'I18N_ABOUT_PAGE_SECTION_SEVEN_TITLE' | translate"></h2>
+    <div class="oppia-about-classroom-button-container-one">
+      <button type="submit" (click)="onClickVisitClassroomButton()"
+              class="btn oppia-about-button oppia-about-button-browse"
+              [innerHTML]="'I18N_ACTION_EXPLORE_LESSONS' | translate">
+      </button>
+    </div>
   </div>
 </div>
+
 <style>
-  .oppia-about-page-container .oppia-about-foundation-tab-anchor {
-    background-color: #00645c;
-    border-radius: 4px;
-    color: white;
-    letter-spacing: 0.7px;
-    width: 33%;
-  }
-  .oppia-about-page-container .oppia-about-foundation-tab-anchor:hover {
-    background-color: #3d9991;
-    text-decoration: none;
-  }
-  .oppia-about-page-container .oppia-about-text-align {
+  .oppia-about-section {
```

Comment:
Organize this in a way such that all the generic classes are on top followed by tag-specific selectors like oppia-about-page-section-one etc.

Reply:
arranged in order such that first we have sections, then common things in sections like title, content, button and then section related selectors.